### DISCUSSION TITLE
improving a categorizer with new data

### DISCUSSION THREAD

**Discussion by delaroob** [umyelab/LabGym#82] [Q&A] [answered]
Hello everyone!

I was just wondering if it was possible to add some training data to an already existing, trained categorizer without having to redo the whole training process? 
(I.e.: I have a categorizer trained on examples from video A, but now I would like to train this categorizer on examples from video B too in order to provide the model with more examples and improve the accuracy without redoing the whole training process) 

Thanks in advance!

**Comment by yujiahu415** [comment, marked as answer]
Hi,

If you want to add more training data into a trained Categorizer, the only way is to combine the new training examples and the existing examples to form a new training dataset, and train a new Categorizer. LabGym generates these stand alone behavior examples that are easy to store or transfer so that you can accumulate and refine the behavior dataset over time.
